#!/usr/bin/env python
PACKAGE = "teamplay"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("defendingStrategy", bool_t, 0, "defending strategy on-off", False)

exit(gen.generate(PACKAGE, "teamplayNode", "teamplayStrategy"))
